From d37fe6af97a45f0567c47932fd5c66d451fcb128 Mon Sep 17 00:00:00 2001
From: Yousong Zhou <yszhou4tech@gmail.com>
Date: Fri, 20 Mar 2020 17:07:03 +0800
Subject: [PATCH] build: tools: fix build failure

Tools build can fail because -stack-protector-all is missing

Signed-off-by: Yousong Zhou <yszhou4tech@gmail.com>
---
 tools/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/Makefile.am b/tools/Makefile.am
index 702fcdd1..7dc70c4e 100644
--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -8,6 +8,7 @@ AM_CPPFLAGS = \
 
 AM_LDFLAGS = \
 	$(LIBRT) \
+	$(WARN_LDFLAGS) \
 	$(NULL)
 
 if !OS_WIN32
